spec/namespaces/cluster.yaml

```diff
@@ -868,11 +868,11 @@ components:
       in: path
       name: name
       description: |-
-        Name of the component template to retrieve.
+        Names of the component templates to retrieve.
         Wildcard (`*`) expressions are supported.
       required: true
       schema:
-        $ref: '../schemas/_common.yaml#/components/schemas/Name'
+        $ref: '../schemas/_common.yaml#/components/schemas/Names'
```

Comment:
`get_component_template` doesn't actually support comma-separated lists, this is something I corrected in this commit: https://github.com/opensearch-project/opensearch-api-specification/commit/7a59510d30200b22792fb0bd5dd827b620030f89

Reply:
@Xtansia, thank you for the review. PHP currently allows users to send name as an array. I'll update the PHP generator to ensure this isn't a breaking change. Thanks again!